### DISCUSSION TITLE
Where should we send fees for creating a credit class?

### DISCUSSION THREAD

**Discussion by ruhatch** [regen-network/regen-ledger#383] [General]
In #351 we introduced a fee for creating a new credit class. In the initial implementation, this fee is simply burned.

We need to make a decision about what to do with these fees in the long-term, so let's create proposals below. I've copied a proposal from @robert-zaremba below.

Once we reach a decision we can open a new issue to implement the new mechanism.

**Comment by ruhatch** [comment]
From @robert-zaremba in #351:

> Here is my proposal for what to do with fees:
> 
> 34% will be burned
> 66% will go to a community pool. Funds from this pool will be used create gitcoin grants for building ecosystem around eco-credits.

**Reply by aaronc** [reply]
Can you explain the rationale for this allocation?

**Reply by robert-zaremba** [reply]
1. Token burn will create a sink mechanism. In general it's good to balance token issuance.
2. creating community pool should have a mission to support credit classes. 

Both use-cases are very important. I propose to allocate more to community pool (2/3) because it will drive and sponsor credit class adoption.

**Reply by alphaBiota** [reply]
I agree that fees for creating new EcoService Credit Classes should be reinvested into the regenerative projects/land stewards holding/receiving EcoService Credits.

However, I imagine that the creation of new EcoService Credit Classes would be a flattening logarithmic curve, with fewer new classes being created over time. But the curve for transferring/applying/granting the actual EcoService Credits would be linear, or even exponential.

**Comment by robert-zaremba** [comment]
## Implementing a pool

We don't have a design for the pool. Most likely it will be a DAO. Until we will have it done we can keep the funds locked in an account managed by the foundation, and once the pool will be ready we will transfer them.
